frontend/.gitignore

```diff
@@ -0,0 +1,42 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
You can kill that file... we have a gitignore in the root and dont need all of this